### ISSUE TITLE
Accept error: accept tcp [::]:8080: accept4: too many open files; retrying in 1s

### ISSUE THREAD

**Issue by Kaesewuerfel** [pterodactyl/panel#3270]
**Background (please complete the following information):**
* Panel or Daemon: 1.3.2
* Version of Panel/Daemon:  1.3.2
* Server's OS: Debian 10
* Your Computer's OS & Browser: Chrome, Firefox, Opera and Edge

**Describe the bug**
Please provide additional information too, depending on what you have issues with:
Panel: PHP 7.4.16
Daemon: `4.19.0-16-amd64 #1 SMP Debian 4.19.181-1 (2021-03-19) x86_64 GNU/Linux` and ` Kernel Version: 4.19.0-16-amd64, Docker: 20.10.6` (your kernel version and information regarding docker)

**To Reproduce**
1. After about 24 hours the error starts

**Expected behavior**
After about 24 hours Wings is no longer accessible via the panel and in the browser console Wings responds with 504.

The following errors are in the Wings log:
```
2021/04/17 13:02:28 http: Accept error: accept tcp [::]:8080: accept4: too many open files; retrying in 1s
 
WARN: [Apr 17 13:09:53.845] failed to update fs disk usage from within routine error=server/filesystem: directorysize: failed to walk directory: open /var/lib/pterodactyl/volumes/4e141b38-3513-4962-b5b9-c475a$

Stacktrace:
open /var/lib/pterodactyl/volumes/4e141b38-3513-4962-b5b9-c475a70a5ee6: too many open files
server/filesystem: directorysize: failed to walk directory
github.com/pterodactyl/wings/server/filesystem.(*Filesystem).DirectorySize
        /home/runner/work/wings/wings/server/filesystem/disk_space.go:193
github.com/pterodactyl/wings/server/filesystem.(*Filesystem).updateCachedDiskUsage
        /home/runner/work/wings/wings/server/filesystem/disk_space.go:144
github.com/pterodactyl/wings/server/filesystem.(*Filesystem).DiskUsage.func1
        /home/runner/work/wings/wings/server/filesystem/disk_space.go:115
runtime.goexit
        /opt/hostedtoolcache/go/1.16.2/x64/src/runtime/asm_amd64.s:1371
```

I have already raised the open file limit significantly, but this is not a nice solution for it.

I have been using Pterodactyl for a very long time now and I never had this problem, even with significantly more users and active servers than now.

If you need more information from me, feel free to let me know.

**Comment by DaneEveritt** [comment]
Much like all of the other issues related to things like this, telling me the only way to reproduce is to leave things running for 24 hours is not going to get this fixed unfortunately.

While I would not be surprised if there is potentially a resource leak somewhere, I'm going to need far more specific environment setup details: how many servers, what types, what other actions are happening on the system, what _are_ the limits you set.

I don't have the type of development time to wait for days to reproduce potential issues, and I don't have a development environment that can facilitate running tens of hundreds of servers at once and be doing all sorts of different actions on them.

Basically, I don't really know _what_ I need to try and debug this, but I can tell you there isn't nearly enough here to be able to help you more than saying "well that kinda sucks" unfortunately.

**Comment by NickEvansDev** [comment]
a quick *temporary* fix is to just restart wings with cron, so it's not running for more than 24 hours a time

**Comment by sanegaming** [comment]
This is starting to happen to me almost daily now. What information would you like me to pull so I can help you guys debug this?
